### ISSUE TITLE
Exception after import-jdl

### ISSUE THREAD

**Issue by shwabnitz** [hipster-labs/generator-jhipster-entity-audit#138]
When creating a fresh jhipster app with entity audit I get directly an exception. Do I make a mistake here? Thanks for help.

- `npm install -g hipster-labs/generator-jhipster-entity-audit#master`
- Created jhipster 6.6.0 application without elasticsearch
- `yo jhipster-entity-audit default`
- `jhipster import-jdl <jdl-file>`

The last command throws the following exception:

```
Running post run module hooks

events.js:174
      throw er; // Unhandled 'error' event

 TypeError: this.getJhipsterAppConfig is not a function
    at module.exports.readConfig (/home/data/dev/projects/audit1/node_modules/generator-jhipster-entity-audit/generators/entity/index.js:13:33)
    at Object.<anonymous> (/home/data/dev/projects/audit1/node_modules/yeoman-generator/lib/index.js:424:27)
    at /home/data/dev/projects/audit1/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /home/data/dev/projects/audit1/node_modules/run-async/index.js:24:19
    at self.env.runLoop.add.completed (/home/data/dev/projects/audit1/node_modules/yeoman-generator/lib/index.js:425:13)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
Emitted 'error' event at:
    at Immediate.setImmediate (/home/data/dev/projects/audit1/node_modules/yeoman-generator/lib/index.js:433:22)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)`
```

Using the following environment:
openjdk 11.0.5
npm version 6.13.6
yo version 3.1.0
node version 10.18.1
jhipster 6.6.0
Entity Audit Generator: 3.1.1
Ubuntu 18.04

**Comment by shwabnitz** [comment]
The following order works:

```
yo jhipster-entity-audit default
jhipster import-jdl <jdl-file>
```

**Comment by murdos** [comment]
@shwabnitz : I released version 3.2.0 with full compatibility with JHipster 6. Could you try again with it and confirm that you don't have anymore the error?

**Comment by shwabnitz** [comment]
@murdos Excellent, both orders work! Thank you.

Closing.
